people/feeback.md

```diff
@@ -0,0 +1,64 @@
+# Feedback
+
+> [Giving feedback turns out to be the unnatural atomic building block atop which the unnatural skill set of management gets built](https://a16z.com/2012/10/17/making-yourself-a-ceo/)
+
+Feedback is important to maintain a shared understanding of what is going well and what needs improvement.
+
+It promotes sharing of ideas on how to maintain and develop a shared understanding into impactful action.
+
+It is an integral part to any iteratively improving system.
+
+---
+
+## Where will feedback be used
+
+Feedack will be used in 1:1s when necessary, at the end of each quarter in regards to PDRs, and as evidence for promotion and remuneration applications.
+
+---
+
+## What do we collect feedback on?
+
+The how and the what.
+
+We collect feedback across 3 lines of your work, technical, inter, and intra team working.
+
+How you do what you do, how you help others to do what they do, and what support is needed to help make you more effective.
+
+"Intra-team" is the smallest group you work with 1 branch removed from yourself. e.g. The project you're currently working on.
+
+"Inter-team" is any other groups outside of your intra-team.
+
+### Technical
```

Comment:
What is _technical_?

Reply:
As below "technical know-how". 1337 skillz.

Basically - applying your technical skills appropriately. Can't think of a better set of terms. Technical approach?